CONTRIBUTING.md

```diff
@@ -237,6 +237,75 @@ After saving and then exiting the editor, `git log` should show a clean history
 `development` branch.  You can also do `git diff yyy..xxx` to make sure nothing new was dropped.  If
 all goes well, you can submit a PR using `xxx` branch.
 Don't worry, if something goes wrong during the rebase, you an always `git rebase --abort` and start over.
+
+## AMReX Coding Style Guide
+
+### Code Guidelines
+
+AMReX developers should adhere to the following coding guidelines:
+  * Indentations should use 4 spaces, not tabs.
+  * Use curly braces for single statement blocks. For example:
+```cpp
+       for (int n=0; n<10; ++n) {
+           Print() << "Like this!";
+       }
+```
+  or
+```cpp
+       for (int n=0; n<10; ++n) { Print() << "Like this!"; }
+```
+  but not
+```cpp
+
+       for (int n=0; n<10; ++n) Print() << "Not like this.";
+```
+  or
+```cpp
+       for (int n=0; n<10; ++n)
+          Print() << "Not like this.";
+```
+  * Add a space after function definitions and declarations (but
+not when simply calling the function). For example:
```

Comment:
This does not seem to be entirely accurate.  The space is in the middle of function definitions and declarations between the function name and `(`.